_apps/commitlint-bot.md

```diff
@@ -0,0 +1,61 @@
+---
+title: commitlint-bot
+description: A GitHub App that lints all commits of new or edited pull requests and
+  sets an appropriate status check.
+slug: commitlint-bot
```

Comment:
We strongly recommend against having the name `bot` in the title of your GitHub App (I know there are already a few published with `bot` in the name, but we're recommending people avoid it going forward). One of the reasons for this is that your app is already labeled a bot by GitHub, so if it comments, it will show up as `commitlint-bot[bot]`.

Reply:
Done :)